code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -244,6 +247,9 @@
 			if(stun_absorption[i]["end_time"] > world.time && stun_absorption[i]["examine_message"])
 				msg += "[t_He] [t_is][stun_absorption[i]["examine_message"]]\n"
 
+	if(just_sleeping)
+		msg += "[t_He] [t_is]n't responding to anything around [t_him] and seem[p_s()] to be asleep.\n"
+
```

Comment:
Could this reasonably use the original code instead of duplicating the message?

Also, maybe make it affect more mobs, *at least* pets.

Reply:
Out of scope for this PR.

> Also, maybe make it affect more mobs, _at least_ pets.

Reply:
I don't know how to do it. There's your answer.

Reply:
```suggestion
	if(just_sleeping)
		msg += "[user.p_they(TRUE)] isn't responding to anything around [user.p_them()] and seems to be asleep.\n"

```
p_`x`(TRUE) makes it capitalized

Reply:
Won't this cause `"They isn't responding to anything` if their face is obscured?